src/content/platform-integration/ios/setup.md

```diff
@@ -102,84 +134,84 @@ on the Apple Developer site.
 
 Set up each iOS device on which you want to test.
 
-### Configure your physical iOS device {:#attach .no_toc}
+ 1. <h3>Configure your physical iOS device</h3>
 
-1. Attach your iOS device to the USB port on your Mac.
+    1. Attach your iOS device to the USB port on your Mac.
 
-1. On first connecting an iOS device to your Mac,
-   your device will display the **Trust this computer?** dialog.
+    1. On first connecting an iOS device to your Mac,
+       your device displays the **Trust this computer?** dialog.
 
-1. Click **Trust**.
+    1. Click **Trust**.
 
-   ![Trust Mac][]
+       ![Trust Mac](/assets/images/docs/setup/trust-computer.png)
+ 
+ 1. <h3>Configure your physical iOS device</h3>
 
-### Enable Developer Mode {: .no_toc}
+    Apple requires enabling **[Developer Mode][]**
+    on the device to protect against malicious software.
 
-Apple requires enabling **[Developer Mode][]**
-on the device to protect against malicious software.
+    1. Tap on **Settings** <span aria-label="and then">></span>
+       **Privacy & Security** <span aria-label="and then">></span>
+       **Developer Mode**.
 
-1. Tap on **Settings** <span aria-label="and then">></span>
-   **Privacy & Security** <span aria-label="and then">></span>
-   **Developer Mode**.
+    1. Tap to toggle **Developer Mode** to **On**.
 
-1. Tap to toggle **Developer Mode** to **On**.
+    1. Restart the device.
 
-1. Restart the device.
+    1. When the **Turn on Developer Mode?** dialog appears,
+       tap **Turn On**.
 
-1. When the **Turn on Developer Mode?** dialog appears,
-   tap **Turn On**.
+ 1. <h3>Create a developer code signing certificate</h3>
 
-### Create a developer code signing certificate {: .no_toc}
+    To send your app to a physical iOS device,
+    _even_ for testing, you must establish trust
+    between your Mac and the device.
+    In addition to trusting the device when that
+    popup appears, you must upload a signed
+    developer certificate to your device.
 
-To send your app to a physical iOS device,
-_even_ for testing, you must establish trust
-between your Mac and the device.
-In addition to trusting the device when that
-popup appears, you must upload a signed
-developer certificate to your device.
+    To create a signed development certificate,
+    you need an Apple ID.
+    If you don't have one, create one.
```

Comment:
Should we make `create one` a link? Maybe to: https://support.apple.com/en-us/108647

Reply:
Great idea, thanks! Updated :)